Runtime/Highlight/DefaultHighlighter.cs

```diff
@@ -0,0 +1,445 @@
+        }
+
+        /// <summary>
+        /// Stops a highlight of given <paramref name="highlightID"/>.
+        /// </summary>
+        public void StopHighlighting(string highlightID)
+        {
+            if (activeHighlights.ContainsKey(highlightID))
+            {
+                activeHighlights.Remove(highlightID);
+            }
+        }
+        
+        /// <summary>
+        /// Regenerates the cached renderers.
+        /// </summary>
+        public void ForceRefreshCachedRenderers()
+        {
+            if (IsHighlighting())
+            {
+                return;
+            }
+            
+            ReenableRenderers();
+
+            if (Application.isPlaying && gameObject.isStatic)
+            {
+                return;
+            }
+            
+            ClearCacheRenderers();
+            RefreshCachedRenderers();
+        }
+
+        protected void ClearCacheRenderers()
+        {
+            renderers = default;
+        }
+        
+        private IEnumerator Highlight(Material highlightMaterial, Func<bool> shouldContinueHighlighting, string highlightID = "")
+        {
+            if (highlightMeshRenderer == null || renderers == null || renderers.Any() == false)
+            {
+                RefreshCachedRenderers();
+            }
+
+            while (shouldContinueHighlighting())
+            {
+                DisableRenders();
+                highlightMeshRenderer.sharedMaterial = highlightMaterial;
+            
+                yield return null;
+            }
+
+            ReenableRenderers();
+
+            if (string.IsNullOrEmpty(highlightID) == false && activeHighlights.ContainsKey(highlightID))
+            {
+                activeHighlights.Remove(highlightID);
+            }
+
+            if (IsHighlighting())
+            {
+                KeyValuePair<string, Material> lastHighlightID = activeHighlights.Last();
+                StartCoroutine(Highlight(lastHighlightID.Value, ()=> IsTheLastKey(lastHighlightID.Key)));
+            }
+        }
+
+        protected void RefreshCachedRenderers()
+        {
+            if (highlightMeshRenderer != null && renderers != null && renderers.Any())
+            {
+                return;
+            }
+            
+            if (highlightMeshRenderer == null)
+            {
+                GenerateHighlightRenderer();
+            }
+            else
+            {
+                highlightMeshRenderer.enabled = false;
+                highlightMeshRenderer.gameObject.SetActive(false);
+            }
+
+            renderers = GetComponentsInChildren<SkinnedMeshRenderer>()
+                .Where(skinnedMeshRenderer => skinnedMeshRenderer.gameObject.activeInHierarchy && skinnedMeshRenderer.enabled)
+                .Concat<Renderer>(GetComponentsInChildren<MeshRenderer>()
+                    .Where(meshRenderer => meshRenderer.gameObject.activeInHierarchy && meshRenderer.enabled)).ToArray();
```

Comment:
```suggestion
                .Where(meshRenderer => meshRenderer.gameObject.activeInHierarchy && meshRenderer.enabled)).ToArray();
```

Reply:
Idk why this keeps happening, I guess Rider...